git-cliff-core/Cargo.toml

```diff
@@ -52,6 +52,7 @@ lazy_static.workspace = true
 thiserror = "1.0.61"
 serde = { version = "1.0.203", features = ["derive"] }
 serde_json = "1.0.118"
+bincode = "2.0.0-rc.3"
```

Comment:
Since the Rc version has been in continuous development, and since it has been in the RC state for quite some time, it works stable. `v1` looks stale (released 3 years ago).
 
There are some breaking changes between the two, and I chose the Rc to avoid needing to upgrade this later once the final tag comes.
https://github.com/bincode-org/bincode/releases

Reply:
Makes sense, thanks for the explanation!